sphinx/writers/latex.py

```diff
@@ -265,6 +265,8 @@ class LaTeXTranslator(nodes.NodeVisitor):
         'classoptions':    '',
         'extraclassoptions': '',
         'passoptionstopackages': '',
+        'geometry':        '\\usepackage[margin=1in,marginparwidth=0.5in]'
```

Comment:
@tk0miya my Python is a bit shaky: should I have used here parentheses or is it ok to have the string split on two lines like here into two substrings?